cmd/kubeadm/app/util/apiclient/wait.go

```diff
@@ -135,7 +135,6 @@ func (w *KubeWaiter) WaitForPodToDisappear(podName string) error {
 // WaitForHealthyKubelet blocks until the kubelet /healthz endpoint returns 'ok'
 func (w *KubeWaiter) WaitForHealthyKubelet(initalTimeout time.Duration, healthzEndpoint string) error {
 	time.Sleep(initalTimeout)
-	fmt.Printf("[kubelet-check] Initial timeout of %v passed.\n", initalTimeout)
```

Comment:
this is a separate commit with a description.

Reply:
omg thank you, this log line has driven me nuts when debugging GCP 😂